fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/PhysicalDataModel.java

```diff
@@ -247,8 +247,8 @@ public void removeTenantPartitions(IDatabaseAdapter adapter, String schemaName,
      * @return
      */
     public ProcedureDef addProcedure(String schemaName, String objectName, int version, Supplier<String> templateProvider,
```

Comment:
If there isn't going to be a split. I suggest changing the name of this to addProcedureAndFunctions

Reply:
Ok, good point, let me change